dart/lib/src/scope.dart

```diff
@@ -14,7 +15,8 @@ class Scope {
   String? transactionName;
 
   /// Returns active transaction or null if there is no active transaction.
-  SentryTransaction? transaction;
+  // SentryTransaction? transaction;
+  ISentrySpan? span;
```

Comment:
doing it like iOS, theres no scope.setTransaction(x) but rather scope.setSpan(x) which is the transaction

Reply:
let's not add the `getLatestActiveSpan` and always return the span bound to the scope, since the UI adjusts the chart timeline by the timestamps, all spans will be children from the root span (bound to the scope)